demos/projects/ST/b-l475e-iot01a/st_code/stm32l4xx_it.c

```diff
@@ -139,15 +139,10 @@ void UsageFault_Handler(void)
 /**
   * @brief This function handles System service call via SWI instruction.
   */
-void SVC_Handler(void)
+/* void SVC_Handler(void)
```

Comment:
what's the point of these newly commented out functions?

Reply:
these handlers are part of port.c in freertos port for cortex-m4 and these files are generated from cubemx. Wrote this comment to make sure how people can pick the file from Cubemx and use.